src/components/two-up-display.tsx

```diff
@@ -67,7 +67,9 @@ export class TwoUpDisplayComponent extends BaseComponent<IProps, IState> {
   private renderRightPanel() {
     const { selectedRightPanel, instructionsIconEnabled, dataIconEnabled, informationIconEnabled } = this.props;
     const title = titles[selectedRightPanel];
-    const contentClass = "content" + (selectedRightPanel === "instructions" ? " scrollable" : "");
+    const contentClass = "content " + (selectedRightPanel === "instructions" ? " scrollable " : "")
+                                    + (selectedRightPanel === "data" ? "resizetofit " : "")
+                                    + this.props.spaceClass;
```

Comment:
These two-up panels are having to handle more and more display cases.  We need to keep our eye on the CSS here to ensure everything displays as expected.